File: C#/TopflytechCodec/TopflytechCodec/Entities/BleTempData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    class BleTempData : BleData
    {
        private float voltage;

        public float Voltage
        {
            get { return voltage; }
            set { voltage = value; }
        }
        private int batteryPercent;

        public int BatteryPercent
        {
            get { return batteryPercent; }
            set { batteryPercent = value; }
        }
        private float temp;

        public float Temp
        {
            get { return temp; }
            set { temp = value; }
        }
        private float humidity;

        public float Humidity
        {
            get { return humidity; }
            set { humidity = value; }
        }
        private Boolean isOpenBox;

        public Boolean IsOpenBox
        {
            get { return isOpenBox; }
            set { isOpenBox = value; }
        }
        private int lightIntensity;

        public int LightIntensity
        {
            get { return lightIntensity; }
            set { lightIntensity = value; }
        }
        private int rssi;

        public int Rssi
        {
            get { return rssi; }
            set { rssi = value; }
        }
    }
}

File: Java/topflytechCodec/src/com/topflytech/codec/entities/BleTempData.java
package com.topflytech.codec.entities;
 

/**
 * Created by admin on 2019/8/29.
 */
public class BleTempData extends BleData{

    public Float getVoltage() {
        return voltage;
    }

    public void setVoltage(Float voltage) {
        this.voltage = voltage;
    }

    public Integer getBatteryPercent() {
        return batteryPercent;
    }

    public void setBatteryPercent(Integer batteryPercent) {
        this.batteryPercent = batteryPercent;
    }

    public Float getTemp() {
        return temp;
    }

    public void setTemp(Float temp) {
        this.temp = temp;
    }

    public Float getHumidity() {
        return humidity;
    }

    public void setHumidity(Float humidity) {
        this.humidity = humidity;
    }


    public Integer getLightIntensity() {
        return lightIntensity;
    }

    public void setLightIntensity(Integer lightIntensity) {
        this.lightIntensity = lightIntensity;
    }

    public Integer getRssi() {
        return rssi;
    }

    public void setRssi(Integer rssi) {
        this.rssi = rssi;
    }

    private Float voltage;
    private Integer batteryPercent;
    private Float temp;
    private Float humidity;
    private Integer lightIntensity;
    private Integer rssi;


}
